packages/projects-docs/pages/learn/getting-started/setting-up-repository.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Repositories Setup + Tips & Tricks
+description: This guide is the first step to get started with CodeSandbox for your repo. It’s a walk-through with a bunch of tips and tricks on how to get the most out of CodeSandbox!
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+This guide will help you get started with using CodeSandbox for your repo. It’s a walk-through with a bunch of tips and tricks on how to get the most out of CodeSandbox!
+
+## Create a Team & Import Repository
```

Comment:
Should we call this section: Import Repository into Team account? It seems like the following paragraph is specific to teams.

Reply:
Oh that's a good point... It's really optional, I've updated the content a bit to say that we recommend it, but it's not required.